{"version":"3.8.0","messages":["\nAn error occurred while loading ./spec/dog_spec.rb.\n\u001b[31mFailure/Error: \u001b[0mrequire_relative \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31m../bin/environment\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m\u001b[0m\n\u001b[31m\u001b[0m\n\u001b[31mLoadError:\u001b[0m\n\u001b[31m  Error loading the 'sqlite3' Active Record adapter. Missing a gem it depends on? can't activate sqlite3 (~\u003e 1.3.6), already activated sqlite3-1.4.0. Make sure all dependencies are added to Gemfile.\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.3.1/gems/activerecord-5.2.2/lib/active_record/connection_adapters/sqlite3_adapter.rb:12:in `\u003ctop (required)\u003e'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.3.1/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:291:in `require'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.3.1/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:291:in `block in require'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.3.1/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:257:in `load_dependency'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.3.1/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:291:in `require'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.3.1/gems/activerecord-5.2.2/lib/active_record/connection_adapters/connection_specification.rb:191:in `spec'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.3.1/gems/activerecord-5.2.2/lib/active_record/connection_adapters/abstract/connection_pool.rb:952:in `establish_connection'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.3.1/gems/activerecord-5.2.2/lib/active_record/connection_handling.rb:60:in `establish_connection'\u001b[0m\n\u001b[36m# ./spec/spec_helper.rb:2:in `require_relative'\u001b[0m\n\u001b[36m# ./spec/spec_helper.rb:2:in `\u003ctop (required)\u003e'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.3.1/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:291:in `require'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.3.1/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:291:in `block in require'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.3.1/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:257:in `load_dependency'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.3.1/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:291:in `require'\u001b[0m\n\u001b[36m# ./spec/dog_spec.rb:1:in `\u003ctop (required)\u003e'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.3.1/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:285:in `load'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.3.1/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:285:in `block in load'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.3.1/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:257:in `load_dependency'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.3.1/gems/activesupport-5.2.2/lib/active_support/dependencies.rb:285:in `load'\u001b[0m\n\u001b[36m# ------------------\u001b[0m\n\u001b[36m# --- Caused by: ---\u001b[0m\n\u001b[36m# Gem::LoadError:\u001b[0m\n\u001b[36m#   can't activate sqlite3 (~\u003e 1.3.6), already activated sqlite3-1.4.0. Make sure all dependencies are added to Gemfile.\u001b[0m\n\u001b[36m#   /usr/local/rvm/gems/ruby-2.3.1/gems/activerecord-5.2.2/lib/active_record/connection_adapters/sqlite3_adapter.rb:12:in `\u003ctop (required)\u003e'\u001b[0m\n","No examples found."],"examples":[],"summary":{"duration":0.000424147,"example_count":0,"failure_count":0,"pending_count":0,"errors_outside_of_examples_count":2},"summary_line":"0 examples, 0 failures, 2 errors occurred outside of examples"}